frog/responsive-images.rkt

```diff
@@ -0,0 +1,117 @@
+#lang racket/base
+
+(require net/url
+         racket/contract/base
+         racket/contract/region
+         racket/file
+         racket/function
+         racket/list
+         racket/path
+         racket/port
+         racket/system
+         racket/string
+         "params.rkt"
+         "util.rkt"
+         "verbosity.rkt"
+         (only-in srfi/1 list-index))
+
+(provide make-responsive clean-resized-images magick-available?)
+
+(define magick-available? (find-executable-path "magick"))
+
+(module+ test
+  (require rackunit))
+
+(define (image-width path)
```

Comment:
The idea was to abstract it should there be another implementation than ImageMagick.. And I made use of this for the pure Racket backend, but now that we decided to scrap it I guess I might as well remove this indirection.